app/lib/nul/authority_records.ex

```diff
@@ -22,6 +22,13 @@ defmodule NUL.AuthorityRecords do
     |> Repo.all()
   end
 
+  def with_stream(func) do
+    Repo.transaction(fn ->
+      stream = Repo.stream(AuthorityRecord)
+      func.(stream)
+    end)
```

Comment:
Might be a good idea to add a `timeout: :infinity` (or some sizable number of seconds) here in anticipation of the day this table gets too big to stream out in <15 seconds. I realize we're at least an order of magnitude away from that in prod, but we might as well be proactive about it.

Reply:
I debated that one internally, and landed on the side of sticking with the default because of the difference in record counts. Don't mind switching it though if your opinion is stronger than mine. 😄

Reply:
I think it's fine, but we'll want to watch the record count in prod and jump on it if it starts to get close. Your demo export took ~12 seconds, so 300,000 is probably a good benchmark.

Reply:
That's what I was thinking too. Plus, if someone notices a download failed because of the default timeout than it'll just be a one-liner change too.